README.md

```diff
@@ -43,3 +43,29 @@ is already set. If you want another registry flavor (i.e. `conservative`) this
 should be defined in the `env:` section of the relevant workflow or step. See
 [Registry flavors](https://pkgdocs.julialang.org/dev/registries/#Registry-flavors)
 for more information.
+
+### Adding Local Registries
+
+Personal registries, e.g. created with [LocalRegistry.jl](https://github.com/GunnarFarneback/LocalRegistry.jl), can be added to the CI using the `localregistry` input option. If the personal registry as well as packages needed in the current project are public, no additional setup is required if the registry url is specified in https-format.
+
+If the registry contains private packages, or is itself private, the ssh protocol should to be used. The user has to provide the corresponding private SSH-keys to the `ssh-agent` to access packages and registry. This can be conveniently done using the [webfactory/ssh-agent](https://github.com/webfactory/ssh-agent) action. A snippet illustrating the usage of (private) personal registries is shown below
+
+```yaml
+...   
+      # Adding private SSH keys (only necessary for accessing private packages and/or 
+      # when providing Registry-link in ssh format)
+      - uses: webfactory/ssh-agent@v0.8.0
+        with:
+          ssh-private-key: |
+            ${{ secrets.PRIVATE_DEPLOY_KEY }}
+            ${{ secrets.PRIVATE_DEPLOY_KEY2 }}
+      - uses: julia-actions/julia-buildpkg@main # Update @main once new taged version available
```

Comment:
```suggestion
      - uses: julia-actions/julia-buildpkg@v1
```

Let's tag a release immediately after merging, so we won't have to update this afterwards